test/cljc/spec_tools/swagger/core_test.cljc

```diff
@@ -239,7 +239,59 @@
               ::swagger/parameters
               {:query (s/keys :opt-un [::name ::street ::city])
                :path (s/keys :req [::id])
-               :body ::address}}))))
+               :body ::address}})))
+    (is (= {:parameters [{:in "query"
+                          :name "name2"
+                          :description "this survives the merge"
+                          :type "string"
+                          :required true}
+                         {:in "query"
+                          :name "name"
+                          :description ""
+                          :type "string"
+                          :required false}
+                         {:in "query"
+                          :name "street"
+                          :description ""
+                          :type "string"
+                          :required false}
+                         {:in "query"
+                          :name "city"
+                          :description ""
+                          :type "string"
+                          :required false
+                          :enum [:tre :hki]
+                          :allowEmptyValue true}
+                         {:in "path"
+                          :name "spec-tools.swagger.core-test/id"
+                          :description ""
+                          :type "string"
+                          :required true}
+                         {:in "body",
+                          :name "spec-tools.swagger.core-test/address",
+                          :description "",
+                          :required true,
+                          :schema {:type "object",
+                                   ;; :title "spec-tools.swagger.core-test/address",
```

Comment:
Is this intentional? If spec is used instead of spec records, the `:title` is present.

Reply:
I think the title should be present in both.